pkg/networkmanager/firewall-client.es6

```diff
@@ -120,6 +121,12 @@ firewalld_dbus.subscribe({
     firewall.dispatchEvent('changed');
 });
 
+cockpit.spawn(['sh', '-c', 'pkcheck --action-id org.fedoraproject.FirewallD1.all --process $$ -u 2>&1'])
```

Comment:
A bit ugly indeed, but I can't think of a better way. If we encounter this more often in the future, we might instead query/store  the bridge pid as a global variable somewhere.

Reply:
I agree. If we start doing this in more places, we probably also want (internal) library functions for that.